quickstarts/maven/docker-file-provided/pom.xml

```
@@ -147,6 +147,7 @@
                                     <name>jkube/context-and-assembly</name>
                                     <build>
                                         <contextDir>${project.basedir}</contextDir>
+                                        <dockerFile>${project.basedir}/src/main/assembly-dockerfile/Dockerfile</dockerFile>
```

This example showcases the use case of a docker file provided with just the contextDir configuration tag. Adding a dockerFile configuration defeats the purpose.
Example should work just by providing this setting:
```suggestion
                                        <contextDir>${project.basedir}/src/main/assembly-dockerfile</contextDir>
```